README.md

```diff
@@ -12,7 +12,8 @@ Variant's basic building blocks are:
 - `variant<Ts...>` - a type-safe representation for sum-types / discriminated unions
 - `recursive_wrapper<T>` - a helper type to represent recursive "tree-like" variants
 - `apply_visitor(visitor, myVariant)` - to invoke a custom visitor on the variant's underlying type
-- `get<T>()` - a function to directly unwrap a variant's underlying type
+- `bool is<T>()` - a function to test for a specific underlying type `T`
+- `T get<T>()` - a function to directly unwrap a variant's underlying type
```

Comment:
@anandthakker - thanks for the PR! I agree with @ricardocosme and @daniel-j-h  - it'll be better to have a separate page listing all available methods while `README` only mentions basic building blocks.